src/Tracker/utils/index.js

```diff
@@ -0,0 +1,45 @@
+export const sendAnalytics = (payload) => {
+  const request = new XMLHttpRequest()
+  request.open('POST', 'https://analytics-sdk-dev.onfido.com/v1/import')
+  request.setRequestHeader('Content-Type', "application/json")
+  request.setRequestHeader('Authorization', "Basic M002Z3tQWURwWTlUanlDKDNRTHIqTDZOQ1hVY2FLZUU6")
+
+  request.onload = () => {
+    if (request.status === 200 || request.status === 201) {
+      console.log(request.response)
+    }
+    else {
+      console.log(request)
+    }
+  }
+  request.onerror = () => console.log(request)
+
+  request.send(payload)
+}
+
+export const formatAnalytics = (woopra, eventName, properties) =>
+  JSON.stringify({
+    batch: [{
+      anonymousId: woopra.cookie,
+      channel: "Web SDK",
+      context: {
+        app: {
+          name: woopra.instanceName,
+          namespace: woopra.options.domain
+        },
+        library: {
```

Comment:
I can make this optional in the backend... Almost certain that i can

Reply:
It looks like it's already optional